TShockAPI/Commands.cs

```diff
@@ -5648,7 +5739,7 @@ private static void Butcher(CommandArgs args)
 				var npcs = TShock.Utils.GetNPCByIdOrName(args.Parameters[0]);
 				if (npcs.Count == 0)
 				{
-					args.Player.SendErrorMessage("Invalid mob type!");
+					args.Player.SendErrorMessage($"{npcs} is not a valid NPC type!");
```

Comment:
```suggestion
					args.Player.SendErrorMessage($"{args.Parameters[0]} is not a valid NPC type!");
```
`npcs` is an empty list here